plugins/activity-resources/src/components/Activity.svelte

```diff
@@ -54,6 +61,16 @@
         .filter((tx) => tx.txClass === core.class.TxCreateDoc)
         .map((it) => [it.objectClass, it.editable ?? false])
     )
+    filter = new Map()
```

Comment:
I think we need to define a model class ActivityFilter and define individual filters as functions inside it.

So by default it will be:

* All - disable filters. 
* Comments - if enabled will show what Comments filter will approve.
* Attachments - if enabled will show what Attachments filter will approve.

More filters could be added later.